README.md

```diff
@@ -59,7 +59,7 @@ The directory where your Brewfile is located.
 
 ## Dependencies
 
-None.
+* [elliotweiser.osx-command-line-tools](https://galaxy.ansible.com/elliotweiser/osx-command-line-tools/)
```

Comment:
Also, just as an aside... isn't it technically against Apples terms to host OS X boxes for VirtualBox? https://github.com/elliotweiser/ansible-osx-command-line-tools/blob/master/molecule.yml#L13-L22 shows various boxes for different OS releases. I love the idea, but the reason I've never put any of my own boxes on Atlas is the legality...